client/my-sites/plugins-wpcom/plugin-panel.jsx

```diff
@@ -8,13 +8,27 @@ import { getSiteSlug } from 'state/sites/selectors';
 import InfoHeader from './info-header';
 import StandardPluginsPanel from './standard-plugins-panel';
 import PremiumPluginsPanel from './premium-plugins-panel';
+
 import BusinessPluginsPanel from './business-plugins-panel';
+import { defaultBusinessPlugins } from './default-plugins';
+
+/*
+ * Interpolate the given plugin propagating `siteSlug` value
+ */
+const linkInterpolator = siteSlug => plugin => {
+	const { supportLink: link } = plugin;
+	const supportLink = link.replace( '{siteSlug}', siteSlug );
+	return Object.assign( plugin, { supportLink } );
```

Comment:
Careful here - this is a mutating operation and we probably don't want to do that

```js
return Object.assign( {}, plugin, { supportLink } )
```